manifests/m/Microsoft/VC++2013Redist-x64/12.0.30501.0/Microsoft.VC++2013Redist-x64.installer.yaml

```
@@ -19,6 +19,8 @@ Installers:
   Scope: machine
   InstallerUrl: https://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x64.exe
   InstallerSha256: E554425243E3E8CA1CD5FE550DB41E6FA58A007C74FAD400274B128452F38FB8
+  InstallerSwitches:
+    Custom: /norestart
   UpgradeBehavior: install
```

```suggestion
  UpgradeBehavior: uninstallPrevious
```

https://github.com/microsoft/winget-cli/issues/1424#issuecomment-910312191